### PR TITLE
mention that InteractiveUtils is loaded automatically in interactive mode

### LINKED ISSUE #36578: Mention in Julia docs that the REPL loads InteractiveUtils
This is a problem that bites a lot of newcomers, when they get used to all the goodness that the REPL brings, but it suddenly fails when using it in scripts.

I think we should mention in the "The Julia REPL" section of the docs that the REPL automatically loads InteractiveUtils.

### PR DESCRIPTION
**Pull request by tpapp**
I am not sure if it is documented elsewhere. Cf [discussion](https://discourse.julialang.org/t/julia-e-subtypes-real-fails/51991/).

Closes #36578.

### CODE REVIEW

**Comment by knuesel** [comment]
This will close https://github.com/JuliaLang/julia/issues/36578

**Comment by tpapp** [comment]
@knuesel: thanks, I missed that, edited the PR description.

File: stdlib/InteractiveUtils/docs/src/index.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Interactive Utilities
 
+This module is intended for interactive work. It is loaded automaticaly in [interactive mode](@ref command-line-options).
```
Comment (fredrikekre) [inline stdlib/InteractiveUtils/docs/src/index.md:3]:
```suggestion
The functionality documented on this page is defined in the `InteractiveUtils` standard library module. This module is intended for interactive work and it is loaded automatically in [interactive mode](@ref command-line-options), for example when working in the REPL.
```

**Review by StefanKarpinski — APPROVED** [review]

File: stdlib/InteractiveUtils/docs/src/index.md (hunk 1 above)
Comment (tpapp) [reply stdlib/InteractiveUtils/docs/src/index.md:3]: > The functionality documented on this page is defined in the `InteractiveUtils` standard library module.

I would have thought that is apparent from the context, like for all other modules.

**Pull request merged**